src/realm/query_engine.cpp

```diff
@@ -503,7 +503,7 @@ void StringNodeFulltext::_search_index_init()
     if (m_link_map->links_exist()) {
         std::set<ObjKey> tmp;
         for (auto k : m_index_matches) {
-            auto ndxs = m_link_map->get_origin_ndxs(k);
+            auto ndxs = m_link_map->get_origin_objkeys(k);
```

Comment:
nit: `ndxs` probably is not the correct name, since we are calling `get_origin_objkeys(k)`, although it is just a renaming of the same method...

Reply:
Good point